app/globals.css

```
@@ -353,3 +353,41 @@
+
+.display-3d-prism {
+  transform-style: preserve-3d;
+  will-change: transform;
+}
+
+.display-3d-face {
+  transform-style: preserve-3d;
+  backface-visibility: visible;
+  -webkit-backface-visibility: visible;
+  -webkit-font-smoothing: subpixel-antialiased;
+  text-rendering: geometricPrecision;
```

Stylelint (17.14.0)

[error] Expected "geometricPrecision" to be "geometricprecision" (value-keyword-case)

(value-keyword-case)